### ISSUE TITLE
Test top 10 downloads for govt-wide accessibility assessment

### ISSUE THREAD

**Issue by eelzi-ONRR** [DOI-ONRR/nrrd#2666]
Also see  #2676 & #2734

The govt-wide 508 assessment requires:

- Identify your agency's top ten viewed publicly available internet pages within the reporting period. Test each page for full conformance to Section 508 (this will require comprehensive manual testing in addition to automated or a hybrid test process if utilized in your agency). For each page listed, report the number of failures by Section 508 standard/WCAG Success Criterion. Report the number of times each SC failed by page. Provide page name, page description and URL. (*Required)
- Identify your agency's top ten viewed electronic documents posted on your public website within the reporting period. Test each document for full conformance to Section 508 (this will require comprehensive manual testing in addition to automated or a hybrid test process if utilized in your agency). For each document, report the number of failures by Section 508 standard/WCAG Success Criterion. Please also provide the document name, document type, and document description.

According to analytics, we have 8 pages from onrr.gov and 2 from NRRD in the top ten page: ![image.png](https://images.zenhubusercontent.com/63482c003bae1bf4351c6f26/1661d4d3-6996-4077-8d35-5c42294a5dd7) ![image.png](https://images.zenhubusercontent.com/63482c003bae1bf4351c6f26/3838490b-1c8c-4571-99f1-ca3200a73642)

Top document downloads:
 ![image.png](https://images.zenhubusercontent.com/63482c003bae1bf4351c6f26/f1cc0d3a-1e15-4e07-8df3-365ba9924ab4)

**Comment by cthomasONRR** [comment]
@eelzi-ONRR I created a new folder within the 508 Govt-Wide Assessment folder for the "Top 10 doc downloads". Folder [here](https://doimspp.sharepoint.com/:f:/r/sites/NRRD/Shared%20Documents/General/ONRR.gov/508/Govt-Wide%20Assessment/Top%2010%20doc%20downloads?csf=1&web=1&e=Jrgu2E).
- Within this folder, I have an excel file titled "_Top_10_Downloads_Details" that contains notes for each file.
- I saved all 10 files within this folder so you can see the version I tested for accessibility. Several docs are updated regularly so I wanted to make sure to capture the version that was tested.
- I also saved visual evidence that the documents are accessible. These are screen captures for Microsoft docs showing the "no issues" in the checker and the color contract checker. For PDFs, I saved the accessibility report.
- One excel doc (https://onrr.gov/document/IndianDistributeeCodes.xlsx) had one failure with the manual color contrast checker. I'm not sure if this manual check is too detailed for this assessment (it passes the Microsoft checker), but the one action item would be to ask the SMEs to change the color on the heading row.

**Comment by eelzi-ONRR** [comment]
@cthomasONRR This is great! 

The instructions from the assessment ask for a document desription for each one. Would you be able to add those to the spreadsheet? A sentence or two should be sufficient.

**Comment by eelzi-ONRR** [comment]
@cthomasONRR Perfect! I think this is all we need for the assessment. Thank you!
